Mathlib/LinearAlgebra/Dual.lean

```diff
@@ -609,9 +603,9 @@ Any finitely-generated free module (and thus any finite-dimensional vector space
 See `Module.IsReflexive.of_finite_of_free`. -/
 class IsReflexive : Prop where
   /-- A reflexive module is one for which the natural map to its double dual is a bijection. -/
-  bijective_dual_eval' : Bijective $ Dual.eval R M
+  bijective_dual_eval' : Bijective (Dual.eval R M)
```

Comment:
Yeah people have been requesting switching from $ to <| in PR reviews, so since I was at it, I just took the liberty to fix these few. I hope the author @ocfnash is okay with the changes! (In fact this was added directly to mathlib4 in [#4989](https://github.com/leanprover-community/mathlib4/commit/e6733eceaa63e98580cb8b30029664641347ce06#diff-1a470a10fd05f0eedb570bfbbe37c36b1a2472e9dc4c78aa7fc070d33ec2c2d2R591-R593).) As for why I changed it to parentheses instead of <|, it's because I don't see the point of using <| (which consumes one more character) when there's no nested parentheses at all.

Reply:
I’m travelling sans laptop so do what you like 😀

Reply:
The main purpose of my comment was to mention a principle that I think is important to follow: in a feature PR, do not make unrelated cosmetic changes or style improvements.

I hope nobody has been requesting switching `$` to `<|` on unrelated lines not touched by a PR. This is what style PRs are for.

My point was *if* you're going to break this rule and have activities outside [the type of the commit](https://leanprover-community.github.io/contribute/commit.html), then I think it's better if you are conservative about it (hence why I suggested that sticking to an analogous style would be better).

Reply:
I see, thanks for the clarification!